### PR TITLE
ui1branch > tl1revbranch

### PR DESCRIPTION
**Pull request by ui-1**
Tl1revbranch > ui1branch

### CODE REVIEW

File: ias/ifintech/v1/iweb/content/core/comprofile/comprofile.php [hunk 1]
```diff
@@ -95,7 +95,7 @@
         <script src="../../lib/superfish/hoverIntent.js"></script>
         <script src="../../lib/superfish/superfish.min.js"></script>
         <!-- Contact Form JavaScript File -->
-        <script src="../../contactform/contactform.js"></script>
+        <script src="../contactform/contactform.js"></script>
```
Comment (hythamsoliman81) [inline ias/ifintech/v1/iweb/content/core/comprofile/comprofile.php]: wrong path!

File: ias/ifintech/v1/iweb/content/core/comprofile/comprofile.php [hunk 2]
```diff
@@ -17,7 +17,7 @@
         <!-- Main Stylesheet File -->
         <link href="../../css/iascontact.css" rel="stylesheet">
           <?php
-          require_once '../../conf/conf.php';
+          require_once '../conf/conf.php';
```
Comment (hythamsoliman81) [inline ias/ifintech/v1/iweb/content/core/comprofile/comprofile.php]: wrong path!

Comment (ui-1) [reply ias/ifintech/v1/iweb/content/core/comprofile/comprofile.php]: what is wrong with it ?

File: ias/ifintech/v1/iweb/content/core/comprofile/comprofile.php (hunk 1 above)
Comment (ui-1) [reply ias/ifintech/v1/iweb/content/core/comprofile/comprofile.php]: what is wrong with it ?

File: ias/ifintech/v1/iweb/content/core/comprofile/comprofile.php (hunk 2 above)
Comment (hythamsoliman81) [reply ias/ifintech/v1/iweb/content/core/comprofile/comprofile.php]: don't use ../ should be a clear path ....

File: ias/ifintech/v1/iweb/content/core/comprofile/comprofile.php (hunk 1 above)
Comment (hythamsoliman81) [reply ias/ifintech/v1/iweb/content/core/comprofile/comprofile.php]: don't use ../ should be a clear path

**Pull request merged**